pkg/static/login.js

```diff
@@ -205,7 +214,7 @@
             const args = [].join.call(arguments, ": ");
 
             if (!window.CSS || !window.CSS.supports.apply(this, arguments)) {
-                disableLogin(args);
+                disableLogin(args, "bypass");
```

Comment:
This is odd pattern to pass string "bypass" into "bypass" argument. Why not just `true`?

Reply:
and we want this only for css not also for js?

Reply:
Using booleans in function API is usually considered a "code smell" as it's not obvious what's going on.

It's written about in programming books and all over the Internet, but here's a pretty decent blog post on the subject:

https://alexkondov.com/should-you-pass-boolean-to-functions/

Reply:
> JavaScript is still mandatory, as functionality breaks without.

found the answer in your original comment

Reply:
Yeah, there's a possibility of optional JS, but we'd have to write defensively, and I'm pretty sure we don't take those things into account. There's no need to support browsers that are a decade old. And, even if we wanted to, they're wildly insecure and our users shouldn't use such an old browser.

We're slower on the uptake of new JavaScript than new CSS (lately).

Things will definitely be broken without proper CSS support... and the we don't generally adopt something new for a year or two anyway (although PF sometimes does sooner, like in the :is()/:where() case).

So if we're going to add a bypass, then, yes, we'd only want that on the CSS and not the JS, unless everyone changed the way they right their JSX and/or we adjust babel specifically for JS... so I figured it's easier and safer to just allow a bypass if only a CSS check fails. (CSS checks come after JS checks for this reason also.)

In other words: There's a big difference between things being completely broken and stuff not looking right (even sometimes horribly wrong or just missing). Letting people through when things are just off-looking or with messed up layouts is OK, as long as we do warn them that their browser doesn't pass the check. But having a browser fail the check for JS and letting them in to be greeted by a non-functional Cockpit is something else entirely.

Therefore, if we just say "if CSS fails, then you can get to it if you really want to" is probably the most correct approach.